chemprop/train/make_predictions.py

```diff
@@ -365,39 +392,41 @@ def make_predictions(
     :return: A list of lists of target predictions. If returning uncertainty, a tuple containing first prediction values then uncertainty estimates.
     """
     if model_objects:
-        (
-            args,
-            train_args,
-            models,
-            scalers,
-            num_tasks,
-            task_names,
-        ) = model_objects
+        (args, train_args, models, scalers, num_tasks, task_names) = model_objects
     else:
-        (
-            args,
-            train_args,
-            models,
-            scalers,
-            num_tasks,
-            task_names,
-        ) = load_model(args, generator=True)
+        (args, train_args, models, scalers, num_tasks, task_names) = load_model(
+            args, generator=True
+        )
 
     num_models = len(args.checkpoint_paths)
 
     set_features(args, train_args)
 
     # Note: to get the invalid SMILES for your data, use the get_invalid_smiles_from_file or get_invalid_smiles_from_list functions from data/utils.py
-    full_data, test_data, test_data_loader, full_to_valid_indices = load_data(
-        args, smiles
-    )
-
-    if args.uncertainty_method is None and (args.calibration_method is not None or args.evaluation_methods is not None):
-        if args.dataset_type in ['classification', 'multiclass']:
-            args.uncertainty_method = 'classification'
+    full_data, test_data, test_data_loader, full_to_valid_indices = load_data(args, smiles)
+
+    if args.uncertainty_method is not None and args.calibration_method in [
+        "conformal_regression",
+        "conformal_quantile_regression",
+    ]:
+        raise ValueError("Conformal regression is not compatible with an uncertainty method")
```

Comment:
I think I want to make this error flag happen before the calibrator is initialized because I end up overwriting the uncertainty_method in the next block. You're right, if some put --uncertainty_method classification, it wouldn't change the functionality since uncertainty_method is overwritten. But I think it is good to notify the user that the uncertainty_method is incompatible rather than softly overwriting it to something else.